README.md

```diff
@@ -784,6 +785,8 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Viraj Shukla](https://github.com/VirajS00)
   - [Visinigiri Aditya](https://github.com/adityakumar48)
   - [Vishnu](https://github.com/shukl08vk)
+  - [Vishal Chauhan](https://github.com/vish-han)
```

Comment:
Please add your name in line 786, after Viraj Shukla.

Reply:
Ok thanks for review